### DISCUSSION TITLE
Incorrect selected tab after leaving a List page

### DISCUSSION THREAD

**Discussion by bum8hj** [community/community#48543] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Description
It's a minor UI bug where the wrong tab is selected/underlined after you leave the page of one of your lists and change tabs.

### Steps to reproduce
1. Click on the Stars tab _(link should look like ../user?tab=stars)_
![image](https://user-images.githubusercontent.com/27828549/221327639-020a3f97-4298-411d-a8b9-eb2c1fc95fa7.png)

2. Enter a list _(link should look like ../stars/user/lists/list-name)_

~3. While on the list page, click **< Stars** to go back~
3. I believe by now it would have already been broken. Jump to the next step.
![image](https://user-images.githubusercontent.com/27828549/221327791-9a703986-62e6-47ac-96c9-5df4c8eb9d2c.png)

4. Select a tab other than Stars and notice how the Stars tab remains underlined
![image](https://user-images.githubusercontent.com/27828549/221327861-ee518f04-ceae-4933-9986-303633b09447.png)

**Comment by lambdakilo** [comment]
I can reproduce this. Activating lists overall seems to disable all of the top underlinings inside the profile page.
